client/assets/scripts/index.js

```diff
@@ -1,10 +1,8 @@
 const searchBtn = document.getElementById('btn-search');
 const randButton = document.getElementById('btn-random');
 const searchIcon = document.getElementById('search-icon');
-let main = document.querySelector('main');            // Main children will be incremented by 1 by the 'article' tag after clicking searchBtn or searchIcon
-                                                    // I changed to 'let' to be able to change the value, but it's not working
+const main = document.querySelector('main');
```

Comment:
selector ought to be ".main" for querySelector()

Reply:
by doing querySelector('main'), I am grabbing the <main> tag, rather than the .main class

Reply:
I think we targeted the HTML tag `main`, not the class.
Also, I pushed the very final version after appointing you as a reviewer. I'm not sure if you can access that version?
This one is buggy a bit see the comments :)

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector as for here
querySelector selectors work exactly the same as the CSS selectors, so without any prefixes, I am calling the tag

Reply:
Sorry, disregard... You're seeing the latest update:)

Reply:
My mistake. I thought you were trying to use a class. Jonathan said he couldn't get the code to run with the selector though...

Reply:
My mistake I thought you were targeting  a class and not the tag

Reply:
You do realise that you have a class of main on the main tag????